### PR TITLE
Make parse work on iOS without editing build hints

### PR DESCRIPTION
**Pull request by codenameone**

### CODE REVIEW

**Comment by sidiabale** [comment]
Thanks for the PR! Saw this issue come up in an SO question some time back. I'm wondering what would happen with this approach if a user wants to, for whatever reason, use a different version of the parse SDK, will it be replaced or will they have to compile their own version?

**Comment by codenameone** [comment]
It will require a compile if someone wants to use a different version. I'm generally of the mind of making the simple trivial and the hard possible. Replacing the version of parse seems like an edge case most developers don't need. But I did see several people complain about missing this hint. It might be because the doc is completely hidden in the native folder (and packaged into the final app as a result).

File: codenameone_library_appended.properties [hunk 1]
```diff
@@ -6,3 +6,4 @@
 
 # Required libs for Parse iOS native SDK (see also https://parse.com/apps/quickstart#parse_push/ios/native/existing)
 codename1.arg.ios.add_libs=Foundation.framework;AudioToolbox.framework;CFNetwork.framework;CoreGraphics.framework;CoreLocation.framework;QuartzCore.framework;Security.framework;StoreKit.framework;SystemConfiguration.framework;Accounts.framework;Social.framework;libz.dylib;libsqlite3.dylib;Parse.a;Bolts.a
+codename1.arg.ios.pods=,Parse ~> 1.12.0
```
Comment (sidiabale) [inline codenameone_library_appended.properties]: Is the preceding comma on purpose?

**Comment by sidiabale** [comment]
@codenameone Thanks for the clarification as per the implications of adding the build hint here. I definitely agree with making the simple trivial and the hard possible.

**Comment by codenameone** [comment]
Yes. If parse adds a pod and another cn1lib adds a pod they won't collide and would have a comma separating them.

**Pull request merged**